test/verify/check-machines

```diff
@@ -2959,9 +2959,8 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b.wait_in_text("body", "Virtual Machines")
         self.waitVmRow("subVmTest1")
 
-        # Check kernel is booted before we try detaching disks
-        if args["logfile"] is not None:
-            wait(lambda: "Linux version" in self.machine.execute("cat {0}".format(args["logfile"])), delay=3)
+        # Wait for the login promt before we try detaching disks - we need the OS to be fully responsive
+        wait(lambda: "login as 'cirros' user" in self.machine.execute("cat {0}".format(args["logfile"])), delay=3)
```

Comment:
What difference does that change make? If it's `None`, then the cat will just fail and thus the wait() time out. It  seemed more correct the other way, or if you want to enforce it, please add an `assert` first.

Reply:
Discussed on IRC, but FTR: the changed search string absolutely makes sense; but I think the `if` needs to be put back or replaced with an assertion.

Reply:
Ah it's always there when ptyConsole in startVm is not set. aka there is not need to assert. That check was not necessary in the first place. ;)